pkgs/development/python-modules/trino-python-client/default.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+  # build
+, setuptools
+  # required
+, pytz
+, requests
+, tzlocal
+  # optional
+, requests-kerberos
+, sqlalchemy
+, keyring
+  # tests
+, pytestCheckHook
+, httpretty
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  pname = "trino-python-client";
+  version = "0.322.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "trinodb";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Hl88Keavyp1QBw67AFbevy/btzNs7UlsKQ93K02YgLM=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    pytz
+    requests
+    tzlocal
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    httpretty
+    pytest-runner
+    requests-kerberos
+    sqlalchemy
+  ];
+
+  pythonImportsCheck = [
+    "trino"
+  ];
+
+  pytestFlagsArray = [
+    # these all require a running trino instance
+    "--deselect=tests/integration/test_types_integration.py"
+    "--deselect=tests/integration/test_dbapi_integration.py"
+    "--deselect=tests/integration/test_sqlalchemy_integration.py"
+    "-k 'not auth'"
+  ];
+
+  passthru = {
+    optional-dependencies = rec {
+      kerberos = [ requests-kerberos ];
+      sqlalchemy = [ sqlalchemy ];
+      external-authentication-token-cache = [ keyring ];
+      all = kerberos ++ sqlalchemy;
+    };
+  };
+
+  meta = with lib; {
+    description = "Client for the Trino distributed SQL Engine";
```

```suggestion
    changelog = "https://github.com/trinodb/trino-python-client/blob/${version}/CHANGES.md";
    description = "Client for the Trino distributed SQL Engine";
```

Updates need to come with changelog URLs, if they exist. Setting `meta.changelog` makes it easier to programatically arrive at the relevant URL.